eth/gasprice.go

```diff
@@ -176,28 +193,26 @@ func (self *GasPriceOracle) lowestPrice(block *types.Block) *big.Int {
 	return minPrice
 }
 
+// SuggestPrice returns the recommended gas price.
+// The return value may not be modified by the caller.
```

Comment:
Could this restriction be lifted by returning a new big.Int copy of the recommended price? It seems like a odd restriction for a exported function unless it's expected to be called very often where allocations could be a performance / memory bottleneck.